app/bundles/PluginBundle/Integration/AbstractIntegration.php

```diff
@@ -379,6 +385,11 @@ public function getRequiredKeyFields ()
         switch ($this->getAuthenticationType()) {
             case 'oauth2':
                 return array(
+                    'consumer_id'      => 'mautic.integration.keyfield.consumerid',
+                    'consumer_secret'  => 'mautic.integration.keyfield.consumersecret'
+                );
+            case 'oauth2':
```

Comment:
Will this case ever be executed when it is the same as the first one? (`oauth2`)